### ISSUE TITLE
test_driver gives error

### ISSUE THREAD

**Issue by rtonnaer** [sam-xl/scancontrol#16]
After running test_driver.launch with the 3060-25 sensor connected, I get the following error:

```
[ INFO] [1598449930.417684470]: There is 1 scanCONTROL device connected.
[ INFO] [1598449930.418853261]: Interface found: MICRO-EPSILON Optronic GmbH-scanCONTROL 3060-25 -519080050
[FATAL] [1598449930.465501693]: The scanCONTROL device is a undefined type.
Please contact Micro-Epsilon for a newer SDK or update the driver.
[FATAL] [1598449931.466602196]: ScanControlDriver: Initialization failed.
```

After this I downloaded the Linux SDK from Micro Epsilon (version 0.2.4) and followed the installation instructions. This seemed to proceed without errors. If I look in /usr/local/lib I have the following output:

```
drwxr-xr-x  5 root root     4096 aug 26 15:50 ./
drwxr-xr-x 10 root root     4096 feb 10  2019 ../
-rw-r--r--  1 root root  3957586 dec 16  2019 libaravis-0.6.a
-rwxr-xr-x  1 root root     1041 dec 16  2019 libaravis-0.6.la*
lrwxrwxrwx  1 root root       22 dec 16  2019 libaravis-0.6.so -> libaravis-0.6.so.0.0.0*
lrwxrwxrwx  1 root root       22 dec 16  2019 libaravis-0.6.so.0 -> libaravis-0.6.so.0.0.0*
-rwxr-xr-x  1 root root   410048 dec 16  2019 libaravis-0.6.so.0.0.0*
lrwxrwxrwx  1 root root       11 aug 26 15:50 libllt.so -> libllt.so.0*
lrwxrwxrwx  1 root root       15 aug 26 15:50 libllt.so.0 -> libllt.so.0.2.4*
-rwxr-xr-x  1 root root    65096 dec 16  2019 libllt.so.0.2.0*
-rwxr-xr-x  1 root root    70752 aug 26 15:50 libllt.so.0.2.4*
lrwxrwxrwx  1 root root       14 aug 26 15:49 libmescan.so -> libmescan.so.0*
lrwxrwxrwx  1 root root       18 aug 26 15:49 libmescan.so.0 -> libmescan.so.0.2.4*
-rwxr-xr-x  1 root root   123864 dec 16  2019 libmescan.so.0.2.0*
-rwxr-xr-x  1 root root   134144 aug 26 15:49 libmescan.so.0.2.4*
drwxr-xr-x  2 root root     4096 aug 26 15:50 pkgconfig/
drwxrwsr-x  4 root staff    4096 aug 18 13:39 python2.7/
drwxrwsr-x  3 root staff    4096 feb 10  2019 python3.6/
```

Could it be that the old SDK is iterfering with proper functioning of the driver?

**Comment by rtonnaer** [comment]
Sorry for the unclear code blocks. Don`t understand how to insert new lines.

**Comment by dave992** [comment]
The driver was made for the older SDK version, which did not contain the newly added 3xxx line of sensors. So that is indeed why the error is shown. 

Updating the libraries should make sure the device is supported by the libraries, but probably the driver does not yet recognize the sensor line (3xxx). I will have a look at where this is happening and propose a fix for this.

**Comment by dave992** [comment]
Could you try the fix I proposed in #17? If you get compile errors regarding `scanCONTROL30xx_25` and `scanCONTROL30xx_xx` not existing, then the old library files are still used.
